src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -52,51 +70,204 @@ export const PriceDatabaseBenefits: React.FC = () => {
 const Section: React.FC<{
   title: string
   text: string
-  Image: ReactElement
-  imagePosition?: "left" | "right"
-}> = ({ title, text, Image, imagePosition = "left" }) => {
+  jsx: ReactElement
+  jsxPosition?: "left" | "right"
+}> = ({ title, text, jsx, jsxPosition = "left" }) => {
   return (
     <>
-      <Media lessThan="sm">
-        <GridColumns my={4} gridRowGap={[2, 0]}>
-          <Column span={6}>{Image}</Column>
-          <Column span={6}>
-            <Text as="h1" variant="xl">
-              {title}
-            </Text>
-            <Text variant="sm">{text}</Text>
-          </Column>
-        </GridColumns>
+      <Media lessThan="md">
+        <Flex mt={4} />
+        <Text as="h1" variant="xl" mt={2}>
+          {title}
+        </Text>
+        <Flex mb={1} />
+        <Text variant="sm">{text}</Text>
+        <Flex mb={2} />
+        {jsx}
+        <Spacer height={80} />
       </Media>
-      <Media greaterThan="sm">
+      <Media greaterThanOrEqual="md">
         <GridColumns my={4} gridRowGap={[2, 0]}>
-          {imagePosition === "left" && <Column span={6}>{Image}</Column>}
+          {jsxPosition === "left" && <Column span={6}>{jsx}</Column>}
           <Column span={6}>
-            <Text as="h1" variant="xl">
+            <Text as="h2" variant="xl">
               {title}
             </Text>
-            <Text variant="sm">{text}</Text>
+            <Text variant="sm" mt={1}>
+              {text}
+            </Text>
           </Column>
-          {imagePosition === "right" && <Column span={6}>{Image}</Column>}
+          {jsxPosition === "right" && <Column span={6}>{jsx}</Column>}
         </GridColumns>
       </Media>
     </>
   )
 }
 
-const SectionImage: React.FC<{ src: string; srcSet: string; alt: string }> = ({
-  src,
-  srcSet,
-  alt,
-}) => {
+const SectionImage: React.FC<{
+  src: string
+  srcSet: string
+  alt: string
+  artistName?: string
+  artworkName?: string
+}> = ({ src, srcSet, alt, artistName, artworkName }) => {
   return (
     <ResponsiveBox
       aspectWidth={800}
       aspectHeight={660}
       maxWidth={800}
       maxHeight={660}
     >
-      <Image width="100%" height="100%" src={src} srcSet={srcSet} alt={alt} />
+      <Image
+        width="100%"
+        height="100%"
+        src={src}
+        srcSet={srcSet}
+        alt={alt}
+        style={{ objectFit: "cover", alignSelf: "center" }}
+      />
+      {!!artistName && !!artworkName && (
+        <>
```

Comment:
small chore: this fragment seems to be not needed here.

Reply:
oh yes, i refactored this and forgot to update it. I will fix it now. Thanks